ci/configuration.properties

```diff
@@ -0,0 +1,18 @@
+# Baseline image name
+docker.java.baseline.image=adoptopenjdk/openjdk8:latest
```

Comment:
We should add explicit versions and switch to `eclipse-temurin` docker images (successor of AdoptOpenJdk/Adoptium).

Let's craft another properties-file for MongoDB and then we should have everything to prepare the merge.